### PR TITLE
Changes for user login functionality

### PR DESCRIPTION
**Pull request by Tirtha1388**
Changes for user login functionality-
1. Created method for adding new user.
2. Created method for authenticating login user.
3. Password encryption has been included.

### CODE REVIEW

File: src/main/java/com/stickynotes/dao/CreateUserDao.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	public UserDto createUser(UserPojo userPojo){
+		
+		UserDto userDto=new UserDto();
+		
+		UserEntity userEntity=new UserEntity();
+		userEntity.setUserid(userPojo.getUserid());
+		userEntity.setName(userPojo.getName());
+		userEntity.setEmail(userPojo.getEmail());
+		userEntity.setCountry(userPojo.getCountry());
+		userEntity.setPassword(hashPassword(userPojo.getPassword()));
```
Comment (vavs0208) [inline src/main/java/com/stickynotes/dao/CreateUserDao.java]: Use model mapper

File: src/main/java/com/stickynotes/readservice/ReadController.java [hunk 2]
```diff
@@ -34,5 +40,50 @@
...
+			
+		} catch (Exception e) {
+			searchUserDto.setPresent(user.isPresent());
+			searchUserDto.setMessege("User login is unsuccessful");
+		}
+		
+		return searchUserDto;
+	}
+	
+	public boolean match(String passText,String passEncoded){
```
Comment (vavs0208) [inline src/main/java/com/stickynotes/readservice/ReadController.java]: This piece of code belongs to business layer not in the Controller.
Request you to create a new BO layer and do the needful there

File: src/main/java/com/stickynotes/readservice/ReadController.java [hunk 3]
```diff
@@ -34,5 +40,50 @@
...
+	@RequestMapping(value = "/getLoginUsers", method = RequestMethod.POST)
+	@ResponseBody
+	public SearchUserDto getLoginUserDetails(@RequestBody UserPojo userRecord) {
+		
+		SearchUserDto searchUserDto= new SearchUserDto();
+		Optional<UserEntity> user= userRepository.findById(userRecord.getUserid());
+		try {
+			
+			if(user.isPresent()){
+				UserEntity userEntity= new UserEntity();
```
Comment (vavs0208) [inline src/main/java/com/stickynotes/readservice/ReadController.java]: You are directly calling the entity layer inside the controller.
Please do this in DAO/ Business layer and refer it back to the controller

**Review by vavs0208 — CHANGES_REQUESTED** [review]

**Comment by Tirtha1388** [comment]
Closing and creating new pull request

**Pull request closed without merging**